nodes.py

```diff
@@ -712,6 +720,9 @@ def load_taesd(name):
         elif name == "taesd3":
             sd["vae_scale"] = torch.tensor(1.5305)
             sd["vae_shift"] = torch.tensor(0.0609)
+        elif name == "taef1":
+            sd["vae_scale"] = torch.tensor(1.5305)
+            sd["vae_shift"] = torch.tensor(0.0609)
```

Comment:
https://github.com/comfyanonymous/ComfyUI/blob/master/comfy/latent_formats.py#L145

These numbers should match the ones here.

Reply:
Thank you.